infra/gp-grafana-instance/values.yaml

```diff
@@ -0,0 +1,14 @@
+# Default values for gp-grafana-instance.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+grafana:
+  auth:
+    admin:
+      user: "admin"
+      password: "insert some password here"
+
+  datasource:
+    prometheus:
+      token: "xxx-instert-Bearer-Token"
```

Comment:
typo

Reply:
wäre es nicht besser password und token auszukommentieren?
Dann muss man bei argo/helm diese werte setzen und es wird kein default genommen

Reply:
In Argo ignorieren und manuell ergänzen.